spec/controllers/two_factor_authentication/sms_opt_in_controller_spec.rb

```diff
@@ -53,7 +53,7 @@
 
     context 'when loaded while adding a new phone' do
       let(:user) { create(:user) }
-      let(:phone) { Faker::PhoneNumber.cell_phone }
+      let(:phone) { Faker::PhoneNumber.cell_phone_in_e164 }
```

Comment:
ooof we should probably do this one everywhere instead of `cell_phone` if it can produce ambiguous results like it did for that example

Reply:
I think it's mostly in how the parser handles it? `459-010-8499` got parsed into `+14590108499` since it didn't have a country code, and `+45 90 10 84 99` stayed as `+45 90 10 84 99` since it was already formatted with one

Reply:
right it's ambiguous data without a `+1`, becuse it could be `+1` or `+45`